app/org/sagebionetworks/bridge/play/controllers/UploadSchemaController.java

```diff
@@ -36,7 +37,7 @@ public Result createOrUpdateUploadSchema() {
         UserSession session = getAuthenticatedSession(DEVELOPER);
         StudyIdentifier studyId = session.getStudyIdentifier();
         
-        UploadSchema uploadSchema = parseJson(request(), UploadSchema.class);
+        UploadSchema uploadSchema = DynamoUploadSchema.fromJson(requestToJSON(request()));
```

Comment:
This tightly couples UploadSchemaController to DynamoUploadSchema, which completely negates the benefits of having an UploadSchema interface. (This might be moot after we remove the fromJson() method.)